posthog/tasks/usage_report.py

```diff
@@ -74,6 +74,7 @@ class UsageReportCounters:
     event_count_in_period: int
     enhanced_persons_event_count_in_period: int
     event_count_with_groups_in_period: int
+    anonymous_personfull_event_count_in_period: int
```

Comment:
Nit: shouldn't this have one l? "personful"

Reply:
We've used "personfull" in internal communications but I think it's a britishism, so yes changing to `personful` 😄